romeo/asset-contract/contracts/asset.clar

```diff
@@ -5,16 +5,16 @@
 
 ;; traits
 ;;
-(impl-trait 'ST1NXBK3K5YYMD6FD41MVNP3JS1GABZ8TRVX023PT.sip-010-trait-ft-standard.sip-010-trait)
+(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait-ft-standard.sip-010-trait)
```

Comment:
Do these traits are only in Mainnet or can be deployed in Devnet with the same principal address?

Reply:
They can be removed alltogether